### DISCUSSION TITLE
Run Spacy on Intel gpu

### DISCUSSION THREAD

**Discussion by armensanoyan** [explosion/spaCy#7876] [Help: Other Questions]
I have an Intel Iris graphics card. Is there any way to make spacy work on my gpu. The CuPy seems to work only on Cuda, which is NVDIA thing. May be there is any way to do it with opencl or ZLUDA ?

**Comment by polm** [comment]
Sorry for the delayed reply, but I'm not aware of anyone using spaCy with non-Nvidia GPUs. Cupy does have experimental AMD support, though I guess that won't help you. 

You could maybe try this [opencl backend for cupy](https://github.com/fixstars/clpy), though it doesn't seem to be actively maintained.
